libs/parallelism/algorithms/tests/unit/algorithms/countif_tests.hpp

```diff
@@ -77,7 +77,11 @@ void test_count_if(ExPolicy&& policy, IteratorTag)
     diff_type num_items = hpx::count_if(policy, iterator(std::begin(c)),
         iterator(std::end(c)), smaller_than_50());
 
+    diff_type num_items1 = hpx::parallel::count_if(policy,
```

Comment:
This probably should not call the old deprecated algorithm. Same below.

Reply:
Oh ok I'll just remove the tests then.